common/src/address.rs

```diff
@@ -39,6 +39,8 @@ pub const CLICKHOUSE_PORT: u16 = 8123;
 pub const OXIMETER_PORT: u16 = 12223;
 pub const DENDRITE_PORT: u16 = 12224;
 pub const MGS_PORT: u16 = 12225;
+pub const WICKETD_PORT: u16 = 12226;
+pub const WICKETD_ARTIFACT_PORT: u16 = 12227;
```

Comment:
Neat, I was wondering if we had a port registry :)

Note that this isn't strictly speaking the wicketd artifact port, since sled-agent can also serve artifacts on this port. I've been calling it the "installinator artifact port", though I've been wondering if we should just call it the "bootstrap artifact port" or something (since it runs on the bootstrap network).

Reply:
Thanks. Fixed.